lib/main.coffee

```diff
@@ -0,0 +1,44 @@
+{CompositeDisposable} = require 'atom'
+
+module.exports =
+  config:
+    executablePath:
+      type: 'string'
+      title: 'PHP Executable Path'
+      default: 'php' # Let OS's $PATH handle the rest
+
+  activate: ->
+    @regex = '(Parse|Fatal) ' +
+            '(?<error>error):(\\s*(?<type>parse|syntax) error,?)?\\s*' +
+            '(?<message>(unexpected \'(?<near>[^\']+)\')?.*) ' +
+            'in .*? on line (?<line>\\d+)'
+    @subscriptions = new CompositeDisposable
+    @subscriptions.add atom.config.observe 'linter-php.executablePath',
+      (executablePath) =>
+        @executablePath = executablePath
+
+  deactivate: ->
+    @subscriptions.dispose()
+
+  provideLinter: ->
+    path = require 'path'
+    helpers = require('atom-linter')
+    child_process = require 'child_process'
```

Comment:
how about you make it `require('child_process')` here and `require('path')` on it's line